Cargo.toml

```diff
@@ -317,6 +315,15 @@ native-fungible = { path = "./examples/native-fungible" }
 non-fungible = { path = "./examples/non-fungible" }
 social = { path = "./examples/social" }
 
+[workspace.dependencies.aws-config]
+default-features = false
+version = "1.6.3"
+
+[workspace.dependencies.aws-sdk-dynamodb]
+default-features = false
+features = ["rt-tokio", "rustls"]
+version = "1.76.0"
```

Comment:
Why this way vs the usual inline?

Reply:
I had a lot more options here at some point and the inline was pretty unreadable.

`cargo publish` normalizes out the inlines anyway :)